sdk/storage/Azure.Storage.Common/tests/Shared/TestConfigurationsTemplate.xml

```diff
@@ -145,6 +146,25 @@
       <TableServiceSecondaryEndpoint>http://[ACCOUNT]-secondary.table.core.windows.net</TableServiceSecondaryEndpoint>
     </TenantConfiguration>
   </TenantConfigurations>
+
+    <!-- A Premium FileStorage account -->
+    <!-- Replace [ACCOUNT] and [ACCOUNT-KEY] -->
+    <TenantConfiguration>
+      <TenantName>PremiumFileTenant</TenantName>
+      <TenantType>Cloud</TenantType>
+      <AccountName>[ACCOUNT]</AccountName>
+      <AccountKey>[ACCOUNT-KEY]</AccountKey>
+
+      <BlobServiceEndpoint>http://[ACCOUNT].blob.core.windows.net</BlobServiceEndpoint>
+      <QueueServiceEndpoint>http://[ACCOUNT].queue.core.windows.net</QueueServiceEndpoint>
+      <TableServiceEndpoint>http://[ACCOUNT].table.core.windows.net</TableServiceEndpoint>
+      <FileServiceEndpoint>http://[ACCOUNT].file.core.windows.net</FileServiceEndpoint>
+      <BlobServiceSecondaryEndpoint>http://[ACCOUNT]-secondary.blob.core.windows.net</BlobServiceSecondaryEndpoint>
+      <QueueServiceSecondaryEndpoint>http://[ACCOUNT]-secondary.queue.core.windows.net</QueueServiceSecondaryEndpoint>
+      <FileServiceSecondaryEndpoint>http://[ACCOUNT]-secondary.file.core.windows.net</FileServiceSecondaryEndpoint>
+      <TableServiceSecondaryEndpoint>http://[ACCOUNT]-secondary.table.core.windows.net</TableServiceSecondaryEndpoint>
```

Comment:
unrelated, but we can think about using <AccountName> to programaticaly compute these endpoints if they're not provided. That would make config less bloated. (We should keep option to override default endpoints though).

Reply:
Maybe in the future.